bool toTurnOnLaser
bool toTurnOffLaser
bool toReadDistance
---
int32 ExecStatus
float32 distance